chemprop/nn/metrics.py

```diff
@@ -44,18 +44,6 @@
 
 
 class Metric(LossFunction):
```

Comment:
So am I understanding correctly that a user can supply `task_weights` to a `Metric` but that they won't be used? If so, why are you deleting the note in the docstring?

Reply:
Good point, I'll add back a note that task weights aren't used to the Metric docstring.